dataset_listing.tsv

```diff
@@ -72,6 +72,9 @@ ieeg_epilepsy	multiple sessions, tutorial	[link](https://neuroimage.usc.edu/bst/
 ieeg_epilepsyNWB	multiple sessions, tutorial — derivative dataset of `ieeg_epilepsy` showcasing the NWB file format alternative	[link](https://neuroimage.usc.edu/bst/getupdate.php?s=tutorial_epimap_bids)	[@TheChymera](https://github.com/TheChymera)	anat, ieeg	T1w, channels, coordsystem, electrodes, events, ieeg, scans
 ieeg_epilepsy_ecog	multiple sessions, tutorial	[link](https://neuroimage.usc.edu/bst/getupdate.php?s=sample_ecog)	[@ftadel](https://github.com/ftadel)	anat, ieeg	T1w, channels, coordsystem, electrodes, events, ieeg, photo, scans
 ieeg_visual_multimodal			[@irisgroen](https://github.com/irisgroen)	anat, fmap, func, ieeg	T1w, bold, channels, coordsystem, electrodes, epi, events, ieeg, sbref
-genetics_ukbb	multiple tasks, T1w, DTI, BOLD, genetic info		[@cpernet](https://github.com/cpernet)	anat, dwi, fmap, func	FLAIR, T1w, bold, dwi, events, info, magnitude1, phasediff
+genetics_ukbb	multiple tasks, T1w, DTI, BOLD, genetic info		[@cpernet](https://github.com/cpernet)	anat, dwi, func	FLAIR, T1w, bold, dwi, events, info
 motion_dualtask	older and younger participants walking while performing discrimination task		[@sjeung](https://github.com/sjeung)	eeg, motion	channels, eeg, events, motion, scans
 motion_spotrotation	participants rotated heading using full-body motion or joystick	[link](https://openneuro.org/datasets/ds004460)	[@sjeung](https://github.com/sjeung)	eeg, motion	channels, coordsystem, eeg, electrodes, events, motion, scans
+mrs_2dmrsi	2D sLASER MRSI data from 8 subjects	[link](https://zenodo.org/records/7701228)	[@markmikkelsen](https://github.com/markmikkelsen)	anat	T1w
```

Comment:
the mrs datatypes and associated suffixes do not appear yet because those columns are updated by relying on pybids that does not know yet about those new things.

I would suggest finishing updating the table and README once the BEP is merged and we have updated pybids accordingly

Reply:
@Remi-Gau Do you have any time to debug why MRS is now disappearing? I can try to look into it tomorrow.